parser/Parser.yp

```diff
@@ -259,6 +259,7 @@ anonblock:  BLOCK blockargs ';' block END
 ;
 
 capture:    ident ASSIGN mdir       { $factory->capture(@_[1, 3])         }
+        |   SET ident ASSIGN mdir   { $factory->capture(@_[2, 4])         }
```

Comment:
note that SET was using `$factory->set($_[2])` wonder if we should still call it in addition to capture

Reply:
`capture` compiles to a `$stash->set()` call on the result of the `mdir`, and `set` compiles to a `$stash->set()` call on the value (for each pair in the setlist). I may have misunderstood, but I don't think calling it in addition would be helpful.